domain/access/state/permission.go

```diff
@@ -712,7 +717,7 @@ FROM    cloud
 WHERE   name = $M.grant_on
 `
 	case corepermission.Offer:
-		return errors.NotImplementedf("db permission support for offers")
+		return nil
```

Comment:
I When we've go "C" on the users and permissions epics, I think you should take one task from the offers epic, which will be to dual write offers.

Then this can be filled in.

It will probably allow us to coordinate with @wallyworld to migrate offer ACLs with applications too.

Reply:
Sounds good, I've made notes of all this in the jira tasks here https://warthogs.atlassian.net/browse/JUJU-5864